library/build.gradle

```diff
@@ -14,8 +14,8 @@ android {
     defaultConfig {
         minSdkVersion 17
         targetSdkVersion 27
-        versionCode 101
-        versionName "1.0.1"
+        versionCode 100
```

Comment:
not 110..at least not less then last version

Reply:
true